system_test/aest_sync_SUITE.erl

```diff
@@ -383,10 +393,10 @@ net_split_recovery(Cfg) ->
     wait_for_value({height, Length * 7}, [net1_node1, net1_node2, net2_node1, net2_node2],
                     Length * 2 * ?MINING_TIMEOUT, Cfg),
 
-    D1 = request(net1_node1, [v2, 'block-by-height'], #{height => Length * 7}, Cfg),
-    D2 = request(net1_node2, [v2, 'block-by-height'], #{height => Length * 7}, Cfg),
-    D3 = request(net2_node1, [v2, 'block-by-height'], #{height => Length * 7}, Cfg),
-    D4 = request(net2_node2, [v2, 'block-by-height'], #{height => Length * 7}, Cfg),
+    {ok, 200, D1} = request(net1_node1, 'GetBlockByHeight', #{height => Length * 7}),
```

Comment:
Shouldn't these use the `get_block` function instead?

Reply:
No. get_block is remainder of the past and I didn't see why it is used. Just didn't want to remove its uses straight away.

Reply:
What do we do about it? New chore in Pivotal?